viz.yaml

```diff
@@ -198,7 +205,7 @@ visualize:
     location: figures/matthew-water.svg
     visualizer: matthew_map
     depends: ["matthew-counties","matthew-flowlines", "matthew-states", "storm-track", "classifyBins", 
-      'storm-location',"matthew-sites", "discharge-classify", "precip-colors", "precip-breaks", "discharge-sparks"]
+      'storm-location',"matthew-sites", "discharge-classify", "precip-colors", "precip-breaks", "discharge-sparks","matthew-borders"]
```

Comment:
the state borders for the states we used for the counties are here, but I didn't add them to the visualization ...